### PR TITLE
V0.0.x/deploy manifests

### PR DESCRIPTION
**Pull request by Al-Pragliola**
in this PR we achieve the following:

example of deployment on the demo_app internal cluster, after deploy you can visit the mocked demo at http://fury-application-status.demo-app.localdomain/

### CODE REVIEW

File: examples/bases/deployment.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      app: fury-application-status
+  template:
+    metadata:
+      labels:
+        app: fury-application-status
+    spec:
+      containers:
+        - name: fury-application-status
+          # remove :mocked when the BE API is ready to be used
+          image: registry.sighup.io/poc/fury-application-status:mocked
```
Comment (nandajavarma) [inline examples/bases/deployment.yml:24]: I would say it is ready to be used now. We have released version 0.1.0 of application status app. :)

**Review by nandajavarma — APPROVED** [review]
Other than fixing the image name everything looks AOK! Lgtm! :-)

File: examples/bases/deployment.yml (hunk 1 above)
Comment (Al-Pragliola) [reply examples/bases/deployment.yml:24]: I should have been more clear in the comment, the FE is not ready yet to consume the api ^^"

Comment (nandajavarma) [reply examples/bases/deployment.yml:24]: clear now! resolving this

**Pull request merged**